drivers/clock_control/stm32_ll_clock.c

```diff
@@ -25,6 +25,54 @@
 #define apb2_prescaler(v) _apb2_prescaler(v)
 #endif /* CONFIG_SOC_SERIES_STM32F0X  */
 
+#define _mco1_prescaler(v) LL_RCC_MCO1_DIV_ ## v
+#define mco1_prescaler(v) _mco1_prescaler(v)
+
+#define _mco2_prescaler(v) LL_RCC_MCO2_DIV_ ## v
+#define mco2_prescaler(v) _mco2_prescaler(v)
+
+#if CONFIG_CLOCK_STM32_MCO1_SRC_NOCLOCK
+	#define LL_RCC_MCO1SOURCE		LL_RCC_MCO1SOURCE_NOCLOCK
```

Comment:
"LL_RCC_MCO1SOURCE": You could not add your own defines using STM32Cube namespace (LL_RCC), unless it is a redefinition of existing define (for a good reason).
We should keep clear separation between STM32Cube definitions and the one created for Zephyr driver.
"MCO1_SOURCE" should be sufficient.